gd32e10x/cmsis/gd/gd32e10x/include/gd32e10x.h

```diff
@@ -49,17 +49,8 @@ OF SUCH DAMAGE.
 #endif /* undefine GD32E10X tip */
 
 /* define value of high speed crystal oscillator (HXTAL) in Hz */
-#if !defined  HXTAL_VALUE
-  #ifdef GD32E103V_EVAL
-  #define HXTAL_VALUE    ((uint32_t)8000000) /*!< value of the external oscillator in Hz */
-  #define HXTAL_VALUE_8M  HXTAL_VALUE
-  #elif defined(GD32E103R_START)
-  #define HXTAL_VALUE    ((uint32_t)25000000) /*!< value of the external oscillator in Hz */
-  #define HXTAL_VALUE_25M  HXTAL_VALUE
-  #else
-  #error "Please select the target board type used in your application (in gd32e10x.h file)"
-  #endif
-#endif /* high speed crystal oscillator value */
+#define HXTAL_VALUE_8M    ((uint32_t)8000000)
+#define HXTAL_VALUE       HXTAL_VALUE_8M /*!< value of the external oscillator in Hz */
```

Comment:
why is this change required? please, check https://github.com/zephyrproject-rtos/zephyr/pull/34970/commits/a93908563631ddba6308587e77cffd70bde30411 on how to solve this properly